### PR TITLE
Update byrow doc.jl for stdze

### PR DESCRIPTION
**Pull request by dyeeee**
Added descriptions and examples for byrow(stdze).

### CODE REVIEW

**Comment by sl-solution** [comment]
Please modify `byrow(ds::AbstractDataset, stdze!, cols; [threads])` to `byrow(ds::Dataset, stdze!, cols; [threads])`

Because stdze! replaces the original data set, thus, it cannot work on view of datasets.

File: src/byrow/doc.jl [hunk 1]
```diff
@@ -1222,11 +1222,69 @@ Variant of `byrow(sort!)` which pass a copy of `ds` and leave `ds` untouched.
...
+   1 │  missing        3.7         9
+   2 │        2       -2.4         4
+   3 │  missing  missing     missing 
+
+julia> byrow(ds,stdze!,:)
+3×3 Dataset
+Row │ x               y               z              
+    │ identity        identity        identity       
+    │ Float64?        Float64?        Float64?       
+─────┼────────────────────────────────────────────────
```
Comment (sl-solution) [inline src/byrow/doc.jl:1282]: this line is not aligned - copy and paste directly from REPL to keep the original format of outputs

**Review by sl-solution — CHANGES_REQUESTED** [review]

**Comment by dyeeee** [comment]
> Please modify `byrow(ds::AbstractDataset, stdze!, cols; [threads])` to `byrow(ds::Dataset, stdze!, cols; [threads])`
> 
> Because stdze! replaces the original data set, thus, it cannot work on view of datasets.

What about other special operations with '!', Shall I change them to Dataset?

**Comment by sl-solution** [comment]
> > Please modify `byrow(ds::AbstractDataset, stdze!, cols; [threads])` to `byrow(ds::Dataset, stdze!, cols; [threads])`
> > Because stdze! replaces the original data set, thus, it cannot work on view of datasets.
> 
> What about other special operations with '!', Shall I change them to Dataset?

Yes, please.  probably, I copied and pasted text and that has caused the problem.

File: src/byrow/doc.jl [hunk 2]
```diff
@@ -679,7 +679,7 @@ julia> byrow(ds, select, :, with = byrow(ds, findfirst, :, by = isodd))
  7
 ```
 @@@@fill!@@@@
-    byrow(ds::AbstractDataset, fill!, cols; [with, by = ismissing, rolling = false, threads])
+    byrow(ds::Dataset, fill!, cols; [with, by = ismissing, rolling = false, threads])
```
Comment (sl-solution) [inline src/byrow/doc.jl:682]: please leave this as AbstractDataset, because, `byrow(fill!)` can fill view of Dataset.

**Review by sl-solution — CHANGES_REQUESTED** [review]

**Review by sl-solution — APPROVED** [review]

**Pull request merged**